code/src/utest/plugin/lsttokens/sizeadjustment/AbbTokenTest.java

```diff
@@ -75,15 +75,15 @@ public void testInvalidInputEmpty() throws PersistenceLayerException
 	public void testValidInputs() throws PersistenceLayerException
 	{
 		assertTrue(parse("Nieder�sterreich"));
-		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB));
+		assertEquals("Nieder�sterreich", primaryProf.get(StringKey.ABB_KR));

```

Comment:
Wow, you are having a close look at these. The string was originally "Niederösterreich" but it looks like the ö has been mangled somewhere along the way, possibly by a change in character set.